luxonis_train/attached_modules/metrics/pml_metrics.py

```diff
@@ -0,0 +1,258 @@
+import torch
```

Comment:
Most of the code here is not covered by tests, please add tests for all the metrics. For new architectures we should require 100% coverage before merge.